communication/slack-config/usergroups.yaml

```diff
@@ -290,3 +290,13 @@ usergroups:
       - mrbobbytables
       - nikhita
       - palnabarun
+
+  - name: kubestellar-devs
+    long_name: KubeStellar Development Team
+    description: "Active developers of KubeStellar on #kubestellar-dev"
+    members:
+      - clubanderson
+      - MikeSpreitzer
```

Comment:
@clubanderson for giggles, can you makes this all lowercase?

Reply:
done